relay-general/src/protocol/measurements.rs

```diff
@@ -0,0 +1,171 @@
+use regex::Regex;
+
+use crate::types::{Annotated, Error, FromValue, Object, Value};
+
+lazy_static::lazy_static! {
+    static ref MEASUREMENT_NAME: Regex = Regex::new("^[a-z0-9-._]+$").unwrap();
+}
+
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, ToValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct Measurement {
+    /// Value of observed measurement value
+    #[metastructure(required = "true", skip_serialization = "empty")]
+    pub value: Annotated<f64>,
+}
+
+#[derive(Clone, Debug, Default, PartialEq, Empty, ToValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct Measurements(#[metastructure(skip_serialization = "empty")] pub Object<Measurement>);
```

Comment:
I'm not entirely sure if this has an effect on this level. @untitaker can you confirm? My guess is that this just has to be annotated on the `measurements` key on the event.

Reply:
It doesn't look like it has an effect according to tests.

Removed in https://github.com/getsentry/relay/pull/724/commits/2b1ecc0831bf7848b425b07080ac6e0a111b3b36